cmd/ndm_daemonset/probe/sysfsprobe.go

```diff
@@ -89,9 +89,26 @@ func (cp *sysfsProbe) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevic
 		klog.Errorf("unable to get sysfs device for device: %s, err: %v", blockDevice.DevPath, err)
 		return
 	}
+	pSysFsDevice := sysFsDevice
```

Comment:
Why is this assignment done?

Reply:
Because `sysFsDevice` is still being used to calculate the capacity here - https://github.com/openebs/node-disk-manager/blob/507409ecc94216ddc467a38e84c1899e4228da0c/cmd/ndm_daemonset/probe/sysfsprobe.go#L158
So a reference to the original blockdevice is still needed

Reply:
another way would be to just move the call which requires a reference to the original blockdevice above all the other calls, and reassign `sysFsDevice` after that

Reply:
> another way would be to move the call which requires a reference to the original blockdevice above all the other calls, and reassign sysFsDevice after that

@Z0Marlin , I thnk this method ^^^ will be better. Anything that requires parent device comes after reassining. Also can you rename `psysfsDevice` to `parentSysFSDevice` and the same with `pDev` to `parentDev`

Reply:
done